pr_agent/tools/pr_reviewer.py

```diff
@@ -266,7 +266,10 @@ def _publish_inline_code_comments(self) -> None:
                 self.git_provider.publish_inline_comment(content, relevant_file, relevant_line_in_file)
 
         if comments:
-            self.git_provider.publish_inline_comments(comments)
+            if get_settings().config.git_provider == 'bitbucket':
+                self.git_provider.publish_bitbucket_inline_comments(comments)
+            else:
+                self.git_provider.publish_inline_comments(comments)
```

Comment:
**Suggestion:** Consider refactoring the code to avoid duplicating the `publish_inline_comments` method. You can create a single method that handles both cases, reducing code redundancy and improving maintainability.
```suggestion
            self.git_provider.publish_inline_comments(comments, is_bitbucket=get_settings().config.git_provider == 'bitbucket')
```